docs/learn/web-development/es-modules.md

```diff
@@ -2,11 +2,11 @@
 title: ECMAScript Modules
 eleventyNavigation:
   key: ES Modules
-  parent: Standards-Based
+  parent: Web Development
   order: 60
 ---
 
-All modern browsers support standard es modules, using `import` and `export` statements:
+All modern browsers support standard es modules. These are javascript files using `import` and `export` statements:
```

Comment:
Does it matter that it's still a module _even if_ it has side effects?

Reply:
Then we have to explain side effects too, and it gets a bit wordy. I think it's fine without it.